openedx/features/ucsd_features/apps.py

```diff
@@ -0,0 +1,21 @@
+from django.apps import AppConfig
+from django.conf import settings
+
+
+LOGIN_EVENT_EMITTER = 'openedx.features.ucsd_features.utils.emit_login_event'
+
+
+class UcsdFeaturesConfig(AppConfig):
+    name = 'openedx.features.ucsd_features'
```

Comment:
Proper naming please.

Reply:
This is necessary as `ucsd_features` is in `openedx/features/` and requires the full path to be used.